config/EventGenerator.xml

```diff
@@ -115,6 +115,16 @@ XSecModel                   alg     Yes  Cross section model used at the thread
   </param_set>
 
 
+ <!--
+  EvtLib / EventLibraryInterface
+ -->
+  <param_set name="EventLibraryInterface">
+     <param type="string" name="VldContext"> </param>
+     <param type="int"    name="NModules">   1                                                    </param>
```

Comment:
We need a dedicated inital state appender or the flux driver will never work properly